docs/src/languages/php/extensions.md

```diff
@@ -60,110 +60,111 @@ You can disable those by adding them to the `disabled_extensions` list.
+| apcu             | *   |     | *   | *   | *   | *   | *   | *   | *   |
+| apcu_bc          |     |     |     | *   | *   | *   | *   | *   |     |
+| applepay         |     |     |     | *   | *   |     |     | *   |     |
+| bcmath           |     |     |     | *   | *   | *   | *   | *   | *   |
+| blackfire        | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| bz2              |     |     |     | *   | *   | *   | *   | *   | *   |
+| calendar         |     |     |     | *   | *   | *   | *   | *   |     |
+| ctype            |     |     |     | *   | *   | *   | *   | *   |     |
+| curl             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| dba              |     |     |     | *   | *   | *   | *   | *   | *   |
+| dom              |     |     |     | *   | *   | *   | *   | *   |     |
+| enchant          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| event            |     |     |     |     | *   | *   | *   | *   |     |
+| exif             |     |     |     | *   | *   | *   | *   | *   |     |
+| ffi              |     |     |     |     |     |     |     | *   | *   |
+| fileinfo         |     |     |     | *   | *   | *   | *   | *   |     |
+| ftp              |     |     |     | *   | *   | *   | *   | *   |     |
+| gd               | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| gearman          | *   | *   | *   |     |     |     |     |     |     |
+| geoip            | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| gettext          |     |     |     | *   | *   | *   | *   | *   |     |
+| gmp              | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| http             | *   | *   |     |     |     |     |     | *   |     |
+| iconv            |     |     |     | *   | *   | *   | *   | *   |     |
+| igbinary         |     |     |     | *   | *   | *   | *   | *   | *   |
+| imagick          | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| imap             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| interbase        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| intl             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| ioncube          |     |     |     | *   | *   | *   |     |     |     |
+| json             |     |     | *   | *   | *   | *   | *   | *   | *   |
+| ldap             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| mailparse        |     |     |     | *   | *   | *   |     | *   |     |
+| mbstring         |     |     |     | *   | *   | *   | *   | *   | *   |
+| mcrypt           | *   | *   | *   | *   | *   |     |     |     |     |
+| memcache         | *   | *   | *   |     |     |     |     |     |     |
+| memcached        | *   | *   | *   | *   | *   | *   |     | *   |     |
+| mongo            | *   | *   | *   |     |     |     |     |     |     |
+| mongodb          |     |     |     | *   | *   | *   | *   | *   |     |
```

Comment:
Looks like they have a v8 release: https://docs.mongodb.com/drivers/php

It's fine if that comes in a follow-up since it's not a bundled extension, but we shouldn't forget about it.

Reply:
I have a MR in progress to build it I will update it once ready